### ISSUE TITLE
[Gutenberg] Identify areas of the Travel theme’s CPT single page that can live as discrete theme blocks and document these for future creation.

### ISSUE THREAD

**Issue by postphotos** [xwp/wp-travel-theme#10]

**Comment by miina** [comment]
We went through the single page mockup today with @DavidCramer and were considering these blocks:

1. Location block:
- Would basically just display the location of the post (adventure), perhaps a link to other adventures in this location?;
- Configurable: alignment;

Question: Does this need to be configurable or will the location of the post in the static position all the time? (@jwold ) In this case there's maybe no need for the block at all;

2. Potential block: amp-gallery:
Question: There seems to be a gallery above the single post, however, it looks like it's above the post's heading and out of the content and thus would not be related to the Gutenberg at all -> no block is needed, is this correct?

Question on Reviews: This could be a block, however, it looks like the reviews can be implemented via commenting system and since these don't seem to appear out of the single post's context anywhere else then looks like no block is needed, is this correct?

So basically it looks like the only block for the single post could be "Location", but if that's in a fixed location, it's not needed either. The post content text can be done with the existing blocks.

Thoughts?
cc @postphotos @jwold @westonruter

**Comment by postphotos** [comment]
Hi @miina, based on the above comment, should I generate new issues out for `location` (and potentially `amp-gallery`)? Let me know!

**Comment by miina** [comment]
@postphotos Based on the design it looks like there are no Gutenberg blocks needed for the CPT template for now -- it looks like the gallery is out of the post content and the location can be just taken from terms to display on the post.

Guessing that at least now there are no blocks needed. Thoughts, @ThierryA and @westonruter?

**Comment by ThierryA** [comment]
@miina yes that sounds good. Though, are we still planning on having a field for the price @postphotos?

**Comment by kienstra** [comment]
**Question About Documentation**

Hi @postphotos,
It sounds like this documentation would fit will in a wiki page in this repo. 

What should this documentation cover?

It sounds like "future creation" refers to documenting our steps, so someone could follow them in creating another theme from an [amp-start templates](https://github.com/ampproject/ampstart/tree/master/css/templates).
